### PR TITLE
Bluetooth: host: Add API to run conn CTE response HCI command

### PR DESCRIPTION
**Pull request by ppryga-nordic**
Add host API to allow execution of HCI_LE_Connection_CTE_Response_-
Enable HCI command.

Signed-off-by: Piotr Pryga <piotr.pryga@nordicsemi.no>

### CODE REVIEW

**Comment by ppryga** [comment]
@jhedberg @alwa-nordic could you take a look?

**Review by Thalley — APPROVED** [review]

File: subsys/bluetooth/host/direction.c [hunk 1]
```diff
@@ -767,3 +813,40 @@ int bt_df_conn_cte_rx_disable(struct bt_conn *conn)
 	return bt_df_set_conn_cte_rx_enable(conn, false, NULL);
 }
 #endif /* CONFIG_BT_DF_CONNECTION_CTE_RX */
+
+#if defined(CONFIG_BT_DF_CONNECTION_CTE_RSP)
+static int bt_df_set_conn_cte_rsp_enable(struct bt_conn *conn, bool enable)
+{
+	CHECKIF(!conn)
+	{
```
Comment (jhedberg) [inline subsys/bluetooth/host/direction.c:821]: minor: the { should be at the end of the first line (basically same coding style as for an if-statement)

**Review by jhedberg — CHANGES_REQUESTED** [review]
Looks pretty good. Just one minor coding style comment.

File: subsys/bluetooth/host/direction.c (hunk 1 above)
Comment (ppryga) [reply subsys/bluetooth/host/direction.c]: Thanks, I've corrected it here and in couple other functions also.

Comment (jhedberg) [reply subsys/bluetooth/host/direction.c]: Thanks, however please keep any coding style fixes to existing code in separate commits (they can still be part of the same PR though)

Comment (ppryga) [reply subsys/bluetooth/host/direction.c]: Yeah, will do :-)

**Comment by ppryga** [comment]
Fixed wrong parameter in `bt_hci_cmd_create` call. Was `BT_HCI_OP_LE_CONN_CTE_REQ_ENABLE`, should be `BT_HCI_OP_LE_CONN_CTE_RSP_ENABLE`.

**Review by jhedberg — APPROVED** [review]

**Pull request merged**